### PR TITLE
lance filter API vs regular pyarrow.txt

### PR DESCRIPTION
**Pull request by jirayanaruto**
I have added the main region for  difference between Lance filter API vs regular pyarrow.

### CODE REVIEW

**Review by jirayanaruto — COMMENTED** [review]
changes on difference are made .

File: docs/Lance filter API vs regular pyarrow.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+The Lance project is a Rust-based library that aims to improve the performance and scalability of filtering operations in PyArrow. One of the key features of the Lance filter API is its ability to execute SQL-like WHERE clause expressions on Arrow tables, using the DataFusion engine under the hood.
```
Comment (changhiskhan) [inline docs/Lance filter API vs regular pyarrow.txt:1]: This is not quite correct. Lance is a disk-format and supports predicate pushdown with a similar interface as pyarrow dataset. The difference is that regular pyarrow datasets take Arrow compute expression whereas Lance accepts a SQL string.

**Review by changhiskhan — COMMENTED** [review]
May I ask if this description was auto-generated? The way it reads, it hits a lot of very high level generic concepts of Lance, pyarrow, etc. But none of the details are quite right.

In addition, this should go in one of the markdown files so it's automatically deployed.

Regardless, thanks for the PR!

**Comment by jirayanaruto** [comment]
Sir i am extremely sorry for this , as it was my first contribution so i
went wrong in it !
EXTREMELY SORRY FOR INCONVENIENCE!

**Comment by changhiskhan** [comment]
no problem! thanks for the contribution. Lmk if you'd like me to keep this open.

**Pull request closed without merging**